docs/getting_started/overview.md

```diff
@@ -1,5 +1,9 @@
 # Overview
 
+## Technical Overview
+
+A technical overview on how OpaquePrompts leverages [confidential computing](https://en.wikipedia.org/wiki/Confidential_computing) and [remote attestation](https://www.redhat.com/en/blog/attestation-confidential-computing) is coming soon.
```

Comment:
Do we want to link to an Azure doc instead of Redhat? Maybe https://learn.microsoft.com/en-us/azure/attestation/tpm-attestation-concepts

Reply:
I do like the Redhat blog because it's specific to CC